test/integration-tests/aspnet/IntegrationTests.AspNet/AspNetTests.cs

```diff
@@ -43,4 +44,36 @@ public async Task SubmitsTraces()
             Assert.True(spans.Count >= 1, $"Expecting at least 1 span, only received {spans.Count}");
         }
     }
+
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    [Trait("Containers", "Windows")]
+    public async Task SubmitsMetrics()
```

Comment:
PR #716 added support to OpenTelemetry Protocol mock collector. I think that it should be our default, proffered method to test all metrics, traces and logs.
It will be great to switch this test to OTLP

The test in current state could be moved to a newly created `IntegrationTests.Exporter.Prometheus` project.

With this we will have, both standardized approach to test instrumentations and separately tested newly added exporter.

Reply:
I agree that it would be better just to have a little sample app for testing different exporters if they work at all. It should not test any instrumentations - just some manual metrics.